src/js/views/patients/sidebar/action/action-sidebar.hbs

```diff
@@ -30,10 +30,18 @@
     <div data-form-sharing-region></div>
     {{/if}}
   </div>
+  {{#if hasAttachments}}
+  <div class="u-margin--t-24">
+    <div class="sidebar__attachments">
+      <h3 class="sidebar__heading">{{far "paperclip"}}<span class="u-margin--l-8">{{ @intl.patients.sidebar.action.actionSidebarTemplate.attachmentsHeadingText }}</span></h3>
+      <div data-attachments-region></div>
+    </div>
+  </div>
+  {{/if}}
   {{#unless isNew}}
   <div class="u-margin--t-24">
-    <h3 class="sidebar__heading">{{ @intl.patients.sidebar.action.actionSidebarTemplate.activityHeadingText }}</h3>
-    <div class="u-margin--t-8 sidebar__activity" data-activity-region></div>
+    <h3 class="sidebar__heading">{{far "timeline-arrow"}}<span class="u-margin--l-8"></span>{{ @intl.patients.sidebar.action.actionSidebarTemplate.activityHeadingText }}</span></h3>
```

Comment:
Per the design docs, add an icon to the activity heading text.

![Screen Shot 2022-11-22 at 4 08 52 PM](https://user-images.githubusercontent.com/35355575/203446451-7c9566c6-812c-4a5f-b305-223b6deab5d2.png)